modules/apps/portal-store/portal-store-gcs/src/main/java/com/liferay/portal/store/gcs/GCSStore.java

```diff
@@ -102,6 +107,80 @@ public void addFile(
 		}
 	}
 
+	@Override
+	public String cleanUpDeletedStoreArea(
+		long companyId, int deletionQuota, TemporalAmount temporalAmount,
```

Comment:
Should we called it `minimumDeletionQuota` instead of `deletionQuota` if this value is less or equal to the number of actually deleted files?

Reply:
Adding "minimum" implies that it should always delete n >= minimumDeletionQuota. I keep it ambiguous/generic on purpose because less than that could be deleted depending on the circumstances. `deletionQuota` is a hint, not something that will be enforced.